content/blog/2023/2023-08-28-fairness-adversarial-debiasing.md

```diff
@@ -0,0 +1,59 @@
+
+The Adversarial Debiasing model results show that fairness metrics, particularly Disparate Impact and Statistical Parity Difference, have improved when debiasing. When not using debiasing, the results are very similar to that of the Random Forest model. When using debiasing, their values moved closer to a fair baseline, signifying reduced biases for these metrics. The Disparate Impact moved from 0.294 to 1.051, suggesting that the model's predictions became more balanced regarding favorable outcomes for privileged and unprivileged groups. The Statistical Parity Difference went from -0.180 to 0.006, indicating a near-zero bias between these groups regarding favorable outcomes.
+
+However, this does not mean that all forms of bias were addressed. Equal Opportunity Difference and Average Odds Difference metrics worsened from -0.097 to 0.358 and -0.087 to 0.197, respectively. This suggests that even though the algorithm has been optimized for specific fairness criteria, other biases have emerged or become more pronounced.
+
+Why is this happening?
+
+- Trade-offs in fairness: Addressing one type of fairness sometimes comes at the expense of another. When we optimize for Disparate Impact or Statistical Parity Difference, it may affect how often positive outcomes are correctly predicted for both privileged and unprivileged groups (equal opportunity). Moreover, it could alter the balance between true positive rates and false positive rates for these groups (average odds difference).
+
+- Adversarial Debiasing implementation: The current algorithm implementation aims to make predictions independent of protected attributes, focusing on balancing outcomes. This approach inherently focuses on balancing outcomes (reflected in the Disparate Impact and Statistical Parity Difference metrics). However, the nuances of other fairness measures might not be as effectively addressed.
+
+When we use debiasing, there can also be a slight decrease in accuracy. This happens because the model is now not only trying to be accurate but also fair. This balance is a necessary trade-off we accept when we want to remove bias. However, it is worth noting that in this example, the accuracy remains comparable to the scenario without debiasing.
+
+Next, let us look at the Box Plot widget. We will use it to show the Disparate Impact and Statistical Parity Difference, much like our previous blog approach.
+
+{{< window-screenshot src="/blog_img/2023/2023-08-28-fairness-adversarial-debiasing-box-plot-bias.png" >}}
+
+{{< window-screenshot src="/blog_img/2023/2023-08-28-fairness-adversarial-debiasing-box-plot-debias.png" >}}
+
+From the box plot, we can see that the model with debiasing enabled gives more balanced results between the unprivileged and privileged groups. This is expected because it is precisely what the Disparate Impact and Statistical Parity Difference metrics measure.
```

Comment:
"we can see that the model" - how do we see that? What do we see and what would we see instead if we wouldn't use debiasing. Reference actual values.